src/protocol/relayer.md

```diff
@@ -0,0 +1,61 @@
+# Layer 1 Relayer/Bridge Protocol
+
+The Fuel relayer/bridge protocol is a set of rules that govern the interaction between the Fuel blockchain and the
+Layer 1 (L1) blockchain (e.g. Ethereum).  
+
+The Fuel blockchain can emit messages that will be processed by the smart contract on the L1 blockchain. The smart
+contract on the L1 can also emit events that will be processed by the Fuel blockchain.
+This is used to move assets between the two blockchains.
+
+## Fuel Message Outbox
+
+The message outbox is the set of messages sent to the L1 blockchain from the Fuel blockchain.
+
+## Fuel Event Inbox
+
+The event inbox is the set of events received from the L1 blockchain by the Fuel blockchain.
+
+The block producer will receive the list of events from the L1 by some relayer, and then include both the count and the
+merkle root of the events in the block header.
+
+There are two types of events that can be received from the L1:
+
+1. Messages
+2. Transactions
+
+### Messages
+
+An arbitrary message sent from the L1 to the Fuel blockchain. This can be used to move assets from the L1
+to the Fuel blockchain or send other arbitrary information to the Fuel blockchain.
+
+| name        | type    | description                                                         |
+|-------------|---------|---------------------------------------------------------------------|
+| `sender`    | `bytes[32]` | The identity of the sender of the message on the L1                 |
+| `recipient` | `bytes[32]` | The recipient of the message on the Fuel Blockchain                 |
+| `nonce`     | `bytes[32]` | Unique identifier of message on L1                                  |
+| `amount`    | `uint64`  | The amount of the base asset transfer                              |
+| `data`      | `byte[]`  | Arbitrary message data                                              |
+| `da_height` | `uint64` | The height of the L1 blockchain when this message event was emitted |
```

Comment:
It would be nice to highlight that this fields is derived from Ethereum, not from the `MessageSent` event:

<img width="1086" alt="image" src="https://github.com/FuelLabs/fuel-specs/assets/18346821/a2cc790c-48c5-4a5e-b1f2-66a43eb1e759">

Reply:
Maybe we don't include it at all. We don't include this in the hash either:
![image](https://github.com/FuelLabs/fuel-specs/assets/11023047/fb141cd7-3a4d-4a8c-9083-901f5e520cfd)